src/13_file_io.py

```diff
@@ -4,15 +4,27 @@
 
```

Comment:
The code works fine but if we define a function for it, it would be more readable.